### DISCUSSION TITLE
Bell and Alarm Icons Swapped

### DISCUSSION THREAD

**Discussion by electricityforprogress** [joeycastillo/Sensor-Watch#298] [Q&A]
I had a question, on my blue board when received the alarm indicator was the bell and the hourly chime was the 'signal' icon.  I though that this was super groovy.  When i just built a set of faces using the web editor and loaded to the watch, I see the icons are swapped (bell for the hourly and signal for the alarm).  Did this recently change in the codebase, any advice on 'swapping' the icons back?    I did include the 'alarm' face, would that change the bell/signal behaviour?  Silly question, thanks for your advice.  cheers!  -Sam

**Comment by wryun** [comment]
See: https://github.com/joeycastillo/Sensor-Watch/issues/132 (I liked the old way too!)
